web-docs/intro.md

```diff
@@ -11,8 +10,30 @@ With flagd you can:
 * use context-sensitive rules to target specific users or user-traits
 * perform pseudorandom assignments for experimentation
 * perform progressive roll-outs of new features
-* aggregate flag definitions from multiple sources 
+* aggregate flag definitions from multiple sources
 
 It doesn't include a UI, management console or a persistence layer.
 It's configurable entirely via a POSIX-style CLI.
 Thanks to it's minimalism, it's _extremely flexible_; you can leverage flagd as a sidecar alongside your application, an engine running in your application process, or as a central service evaluating thousands of flags per second.
+
+# How can deploy flagd?
+
+flagd is designed to fit well into a variety of infrastructures, and can run on various architectures.
+It run as a separate process or directly in your application.
+It's distributed as a binary, container image, and various libraries.
+If you're already leveraging containers in your infrastructure, you can extend the docker image with your required configuration.
+You can also run flagd as a service on a VM or a "bare-metal" host.
+If you'd prefer not to run an additional process at all, you can run the flagd evaluation engine directly in your application.
+No matter how you run flagd, you will need to supply it with feature flags.
+The flag definitions supplied to flagd (*sources*) are monitored for changes which will be immediately reflected in flagd's evaluations.
+Currently supported sources include files, HTTP endpoints, Kubernetes custom resources, and proto-compliant gRPC services.
+
+<!-- Should we add some simple diagrams here to illustrate the deployment options and sources? -->
```

Comment:
We could add a Grid section linking out the various deployment sections.

https://squidfunk.github.io/mkdocs-material/reference/grids

I wouldn't recommend adding an image here.

Reply:
I will convert this comment into a TODO comment to link to the deployment scenario as you discrub.